daemon/cmd/daemon.go

```diff
@@ -686,6 +696,26 @@ func NewDaemon(ctx context.Context, cancel context.CancelFunc, epMgr *endpointma
 	if len(restoredCIDRidentities) > 0 {
 		d.ipcache.UpsertGeneratedIdentities(restoredCIDRidentities)
 	}
+	// Upsert restored local Ingress IPs
+	nIngressIPs := 0
+	var firstErr error
+	for _, ingressIP := range oldIngressIPs {
+		_, err := d.ipcache.Upsert(ingressIP.String(), nil, 0, nil, ipcache.Identity{
+			ID:     identity.ReservedIdentityIngress,
+			Source: source.Local,
+		})
+		if err == nil {
+			nIngressIPs++
+		} else if firstErr == nil {
+			firstErr = err
+		}
+	}
+	if firstErr != nil {
+		log.WithError(firstErr).WithField(logfields.Count, fmt.Sprintf("%d", len(oldIngressIPs)-nIngressIPs)).Error("error while restoring ingress identities")
+	}
+	if nIngressIPs > 0 {
+		log.WithField(logfields.Count, fmt.Sprintf("%d", nIngressIPs)).Info("Restored ingress identities")
```

Comment:
nit: Is there a reason for the `fmt.Sprintf()` or could we just pass the integer directly into the logrus call? In the end, the library will end up doing the `fmt.Sprint()` for us, so we shouldn't need to handle that detail here.

Reply:
I forgot that logrus formats types to strings :-)